Gruntfile.js

```diff
@@ -99,6 +111,12 @@ module.exports = function (grunt) {
         "mochaTest",
         "run:testRules",
     ].concat(checkBinTest));
+    grunt.registerTask("docs", [
```

Comment:
Agreed, it can run when a tag is created for a non-dev release. People will still be able to do this manually though if they update other parts of the docs site that aren't generated via this task.

Can we automatically push to our repo from CircleCI?